manifests/charts/istio-cni/Chart.yaml

```diff
@@ -9,6 +9,6 @@ keywords:
   - istio-cni
   - istio
 sources:
-  - https://github.com/istio/istio/tree/master/cni
+  - https://github.com/istio/istio/tree/release-1.22/cni
```

Comment:
I'm not to sure about this being the full source. In the other charts we seem to have:
```
https://github.com/istio/istio
```
which wouldn't need to release updated each release.

Reply:
let's revert this, and fix it in master